### DISCUSSION TITLE
When I recorded WebGL Canvas, after exporting the recorded video, it was playing very fast.

### DISCUSSION THREAD

**Discussion by saurabhburadkar** [amandaghassaei/canvas-capture#26] [General]
When I recorded WebGL Canvas, after exporting the recorded video, it was playing very fast.

**Comment by saurabhburadkar** [comment]
How can I slow it down?

**Comment by amandaghassaei** [comment]
you can change the fps of the recording with:

`CanvasCapture.beginVideoRecord({ fps: 30 });`

default is 60 fps.  Let me know if that fixes it.
